.github/workflows/semver-checks-pr-label.yml

```diff
@@ -0,0 +1,79 @@
+on:   # yamllint disable-line rule:truthy
+  workflow_run:
+    workflows: ["semver"]
+    types: [completed]
+
+name: Check semver breaks - Label and Comment PR
```

Comment:
I think we definitely need to switch to Gerrit.

Does anyone know why the hell this thing does not want to run?
I even changed the name from the other job from `Check semver breaks` -> `semver` to see if GH was getting bogged down by the spaces, but no luck!

Reply:
> Does anyone know why the hell this thing does not want to run?

I'm certain it's because we haven't merged it yet - we would have the same vulnerability otherwise.

I suggest you try to develop the action in your own repo first and then make a PR once it workss.

Reply:
> I will start modifying https://github.com/rust-bitcoin/rust-bitcoin/pull/3003 accordingly.

+1 - I was thinking of trying out semvar also but in a much simpler repo (first).

Reply:
Success it works! I will upstream the patch to this PR.

Action: https://github.com/storopoli/rust-bitcoin/actions/runs/9913075380/job/27389270899

PR labelled: https://github.com/storopoli/rust-bitcoin/pull/9

Reply:
Cool! Made me realize we don't need a label for non-additive cargo features - they are a bug and thus always wrong.